aquasec/resource_host_runtime_policy.go

```diff
@@ -496,7 +494,7 @@ func resourceHostRuntimePolicyRead(ctx context.Context, d *schema.ResourceData,
 		d.Set("windows_registry_monitoring", flattenWindowsRegistryMonitoring(crp.RegistryAccessMonitoring))
 		d.Set("windows_registry_protection", flattenWindowsRegistryProtection(crp.ReadonlyRegistry))
 
```

Comment:
duplicate set id, also in line: 460

Reply:
460 is the create function, this is the read function?